tests/RestApiSuite/TestPost/content.txt

```diff
@@ -1,12 +1,20 @@
 !|post|${URL}/delete/users|
 
-!|post    |${URL}/users|
-|FirstName|LastName    |
-|Tony     |Soprano     |
-|Jennifer |Melfi       |
-
-!|Query:http result as table|${URL}/users|
-|FirstName                  |LastName    |
-|Tony                       |Soprano     |
-|Jennifer                   |Melfi       |
+!|post    |${URL}/users    |
+|FirstName|LastName|IsMafia|
+|Tony     |Soprano |True   |
+|Jennifer |Melfi   |False  |
+
+!|script|RestTools                       |
+|check|get_json_str|${URL}/users|[{"FirstName": "Tony", "IsMafia": true, "LastName": "Soprano", "id": 1}, {"FirstName": "Jennifer", "IsMafia": false, "LastName": "Melfi", "id": 2}]|
```

Comment:
tests are examples of use - so I see here you have introduced a new fixture get_json_str - which is confusing
I suggest to use existing GET fixture plus check RAW result after this get by RAW Result fixture
it also would be better to create separate test for this specific use case